usr/share/rear/prep/BAREOS/default/400_prep_bareos.sh

```diff
@@ -8,3 +8,6 @@ PROGS+=( "${PROGS_BAREOS[@]}" )
 if [[ "$TAPE_DEVICE" || "$BEXTRACT_DEVICE" ]] ; then
     PROGS+=( mt )
 fi
+
+### empty config file filled by "prep" (and used by restore)
+> $VAR_DIR/bareos.conf
```

Comment:
can you please check again which of those setting you put into this file are actually part of the configuration? There is no need to put those into an extra file as ReaR copies over the configuration.

Such a file should be only used for data that must be determined dynamically during the `mkrescue` workflow.